packages/adblocker/src/lists.ts

```diff
@@ -136,13 +137,20 @@ export function f(strings: TemplateStringsArray): NetworkFilter | CosmeticFilter
 export function parseFilters(
   list: string,
   config: Partial<Config> = new Config(),
-): { networkFilters: NetworkFilter[]; cosmeticFilters: CosmeticFilter[] } {
+): {
+  networkFilters: NetworkFilter[];
+  cosmeticFilters: CosmeticFilter[];
+  preprocessors: Preprocessor[];
+} {
   config = new Config(config);
 
   const networkFilters: NetworkFilter[] = [];
   const cosmeticFilters: CosmeticFilter[] = [];
   const lines = list.split('\n');
 
+  const conditions: Map<string, Preprocessor> = new Map();
+  const preprocessors: { instance: Preprocessor; negated: boolean }[] = [];
```

Comment:
how about naming this a `preprocessorStack`?

Reply:
Looks nice to me. What about `conditions`? I don't have a decent naming for that. However, maybe using `preprocessors` for that can be better.